.prettierignore

```diff
@@ -0,0 +1,4 @@
+# Ignore artifacts:
+build
```

Comment:
SLS creates a folder called `.build` which isn't included, you may want to include it (as prettier does try and frisk it) :smile:

Reply:
Same goes for all the `config/*.json` files